app.py

```diff
@@ -27,19 +27,16 @@
 
 load_dotenv()
 openai_api_key = os.getenv("OPENAI_API_KEY")
+langchain_api_key = os.getenv("LANGCHAIN_API_KEY", "")
```

Comment:
how was this working before without the APi key?

Reply:
Ah it's only for pulling private prompts. The prompt we were using was public, so we didn't need the API key. I was testing something out with a private prompt.